### ISSUE TITLE
Aidan's Master List of Issues

### ISSUE THREAD

**Issue by RAidanM** [themeetinghouse/web#798]
Not in any particular order of priority:

**"Upcoming Event" Module**

* [x]  Some parishes are not propelling displaying events (e.g. waterloo)

**Alt Text**

* [x] Teaching Black box often use incorrect alt text
* [x] Alt text for all videos on the teaching page (including kids, youth, adult page videos) are not formatted the same way leading to inconsistencies across all alt text for thumbnails. As well as frequent typos for the alt text of images on the page
* [x] Blog entries displayed on the home page use different alt text than when they are displayed on the blog page
* [x] Some kid videos have a void series attached to them
* [x] Kids & Youth graphics have incorrect or missing alt text

**Screen Reader Accessibility**

* [x] The Search bar does not speak when moused over when using (at least while using NVDA)
* [ ] All Scroll lists do not guide the user on how to navigate them audibly
* [x] Menu items give inconsistent amounts of information when interacted with
* [x] The Apple Podcasts integration have nothing read out to the user (at least while using NVDA)

(Duplicate issues can be removed from the list if found)

**Comment by lucastbelem** [comment]
@itbel can you pls go through this list and work on the quick wins?
